__tests__/app.test.js

```diff
@@ -250,3 +252,66 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("200- successfully INCREMENTS the votes on an article by a given inc_votes value", () => {
+    const voteUpdate = { inc_votes: 15 };
+    return request(app)
+      .patch("/api/articles/13")
+      .send(voteUpdate)
+      .expect(200)
+      .then(({ body: { updatedArticle } }) => {
+        expect(updatedArticle).toEqual({
```

Comment:
I would use a flexible matcher for this instead eg expect.objectContaining or .toMatchObject